168162095 Implement feedback on login

Joe-Joseph opened the pull request:
#### What does this PR do?
 - Implement feedback on signin
#### Description of Task to be completed?
 - Change error messages
 - change the position of <div> that displays error
 - Put the form and the picture in the center
#### How should this be manually tested?
 -  Clone the repo `git clone https://github.com/andela/ah-jawans-frontend.git`
 - Run `npm install` to install dependencies
 - Run `npm run dev`
 - Go this link `http://localhost:8080/login`
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
 - [168162095](https://www.pivotaltracker.com/story/show/168162095)
#### Screenshots (if appropriate)
 
![Login demo](https://user-images.githubusercontent.com/44877921/64008862-28c20100-cb17-11e9-9f1f-0192d94acb6a.gif)


#### Questions:

Commit:
bg(Login): implement feedback on login

- change the position of <div> that displays error
- Put the form and the picture in the center
- Change error messages

[fixes  #168162095]

Commit:
bg(Login): implement feedback on login

- change the position of <div> that displays error
- Put the form and the picture in the center
- Change error messages

[fixes  #168162095]

pngabo reviewed (changes requested):
update you commits from started to Finishes

shaluchandwani commented on src/redux/thunks/index.js:
```diff
@@ -4,10 +4,15 @@ export const axiosInstance = axios.create({
+) => (dispatch) => axiosInstance[method](endpoint, data)
   .then((response) => {
     dispatch(actionCreator(response.data));
+    localStorage.setItem('token', JSON.stringify(data.data));
```
you can remove this line , i am storing the localstorage token in my PR. this is not the correct place to do it.

billkabanga commented:
Good work on this PR @Joe-Joseph , However I notice that your PR title doesnot imply what the PR solves for. Please make your PR titile implicit. Also check your commit message. You `fix` a bug you don't `finish` it. You should have `[fixes ....]` not `[finishes ...]`

Joe-Joseph committed:
Delete FormContainer.js

Joe-Joseph committed:
bg(Login): implement feedback on login

- change the position of <div> that displays error
- Put the form and the picture in the center
- Change error messages

[fixes  #168162095]

Commit:
bg(Login): implement feedback on login

- change the position of <div> that displays error
- Put the form and the picture in the center
- Change error messages

[fixes  #168162095]

Commit:
Squashed commit of the following:

commit 257ebf639f0a8c9ca04f0e88a58c8aa8ca588cd1
Author: Joseph Nkurunziza <josetheblessing@gmail.com>
Date:   Tue Aug 20 11:12:43 2019 +0200

    ft(Login): Allow user to login

    - Design login page
    - Consume Login API
    - Add tests

    [Finishes #166790050]

commit 95a44dc68b717262fdf6ab5efa22b2c335f83575
Author: Patrick Ngabonziza <ngabonzizap7@gmail.com>
Date:   Tue Aug 20 15:13:36 2019 +0200

    ft(signup): user signup to authors haven

     - Design signup page
     - consume api
    [Finishes #166790048]

commit 6c648279b510a2e2ecbf2bcfb96f5a5d227aa569
Author: kagabof <faustinkagabo@gmail.com>
Date:   Mon Aug 19 09:35:48 2019 +0200

    ch(deploying on heroku): Deploy the project to heroku

    - Added procfile for heroku.
    - Added script in package.json

    [delivers #167941952]
bg(Login): implement feedback on login

- change the position of <div> that displays error
- Put the form and the picture in the center
- Change error messages

[fixes  #168162095]

Commit:
bg(Login): implement feedback on login

- change the position of <div> that displays error
- Put the form and the picture in the center
- Change error messages

[fixes  #168162095]
